### DISCUSSION TITLE
How to show `>` beside the items at the left menu?

### DISCUSSION THREAD

**Discussion by AngangGuo** [McShelby/hugo-theme-relearn#519] [Q&A] [answered]
How to set the theme to show `>` beside menu items? 
Please advise.
![image](https://user-images.githubusercontent.com/5448794/236290163-23dbc4a9-fe50-4f86-a7df-c2f67b0c1a88.png)

**Comment by skiboyjcmu** [comment, marked as answer]
@AngangGuo -  in config.toml, set   **collapsibleMenu = true**

If set to true, the menu in the sidebar will be displayed in a collapsible tree view. Although the functionality works with old browsers (IE11), the display of the expander icons is limited to modern browsers
  collapsibleMenu = true

**Comment by AngangGuo** [comment]
It works great. Thanks for your help and the great work.
